subsys/net/ip/ipv4.h

```diff
@@ -231,6 +231,16 @@ static inline void net_ipv4_set_dscp(uint8_t *tos, uint8_t dscp)
 	*tos |= (dscp << NET_IPV4_DSCP_OFFSET) & NET_IPV4_DSCP_MASK;
 }
 
+/**
+ * @brief Convert DSCP value to priority.
+ *
+ * @param dscp DSCP value.
+ */
+static inline uint8_t net_ipv4_dscp_to_priority(uint8_t dscp)
+{
+	return dscp >> 3;
```

Comment:
Where does this bit shift come from?

Reply:
DSCP is 6 bits, so, we map this to buckets of 8, each bucket is 1 priority level. E.g., DSCP=63-56 -> priority 7.